test/unit/tab/index.test.js

```diff
@@ -305,3 +305,25 @@ test('#componentWillUnmount destroys foundation', () => {
   wrapper.unmount();
   td.verify(foundation.destroy(), {times: 1});
 });
+
+test('on focus event calls handleFocus on TabRipple', () => {
+  const wrapper = shallow(<Tab />);
```

Comment:
It looks like you're trying to get the ref here...you can just use `mount` instead of shallow. Then these 3 lines can just be:

```
const wrapper = mount(<Tab />);
const ripple = wrapper.instance(). tabRipple_.current;
```